### DISCUSSION TITLE
Add annotations to all secrets managed by Strimzi

### DISCUSSION THREAD

**Discussion by jesussancheztellomm** [strimzi/strimzi-kafka-operator#5825] [Q&A]
Hello,

I see that you can add custom annotations to the clusterCaCert but i don't know if it's possible to add it do the rest of secrets created by Strimzi (clustername-cluster-ca, clustername-clients-ca-cert, clustername-clients-ca, etc)

Thanks in advance,
All the best

**Comment by scholzj** [comment]
The `clusterCaCert` metadata were made configurable because the public key of the cluster CA is often copied to other namespaces for the Kafka clients to use it and adding annotations or labels to it helps to have other tools pick it up and copy it automatically. There was no such use-case for the other secrets in the past. If you have such use-case, maybe it can be added.
